sysdump/sysdump.go

```diff
@@ -2098,7 +2098,11 @@ func (c *Collector) SubmitTetragonBugtoolTasks(pods []*corev1.Pod, tetragonAgent
 // (e.g. "a/b/c" => "b/c"). Don't pass an absolute path. It doesn't do what
 // you think it does.
 func removeTopDirectory(path string) (string, error) {
-	index := strings.IndexByte(path, filepath.Separator)
+	index := strings.IndexByte(path, '/')
+	// let's try Windows file separator
+	if index < 0 {
+		index = strings.IndexByte(path, '\\')
+	}
```

Comment:
Technically I don't think we need to actually try the windows separator since the sysdump archives will be created on Linux only.

Reply:
Ok, let's simplify it. Removed.